[2023-02-04 07:40:02,312] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.get_rates_rows_cnt scheduled__2023-02-04T07:35:00+00:00 [queued]>
[2023-02-04 07:40:02,321] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.get_rates_rows_cnt scheduled__2023-02-04T07:35:00+00:00 [queued]>
[2023-02-04 07:40:02,322] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 07:40:02,322] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-04 07:40:02,323] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 07:40:02,337] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_rates_rows_cnt> on 2023-02-04 07:35:00+00:00
[2023-02-04 07:40:02,341] {standard_task_runner.py:52} INFO - Started process 6414 to run task
[2023-02-04 07:40:02,343] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'get_rates_rows_cnt', 'scheduled__2023-02-04T07:35:00+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag_test.py', '--cfg-path', '/tmp/tmpnk5jc1tx', '--error-file', '/tmp/tmpxwtb0moy']
[2023-02-04 07:40:02,344] {standard_task_runner.py:80} INFO - Job 153: Subtask get_rates_rows_cnt
[2023-02-04 07:40:02,407] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.get_rates_rows_cnt scheduled__2023-02-04T07:35:00+00:00 [running]> on host 39df186e1726
[2023-02-04 07:40:02,474] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=get_rates_rows_cnt
AIRFLOW_CTX_EXECUTION_DATE=2023-02-04T07:35:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-04T07:35:00+00:00
[2023-02-04 07:40:02,482] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-02-04 07:40:02,498] {logging_mixin.py:115} INFO - 21
[2023-02-04 07:40:02,508] {python.py:173} INFO - Done. Returned value was: None
[2023-02-04 07:40:02,520] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=get_rates_rows_cnt, execution_date=20230204T073500, start_date=20230204T074002, end_date=20230204T074002
[2023-02-04 07:40:02,556] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-04 07:40:02,599] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-02-04 07:40:26,288] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.get_rates_rows_cnt scheduled__2023-02-04T07:35:00+00:00 [queued]>
[2023-02-04 07:40:26,310] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.get_rates_rows_cnt scheduled__2023-02-04T07:35:00+00:00 [queued]>
[2023-02-04 07:40:26,310] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 07:40:26,311] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-04 07:40:26,311] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 07:40:26,341] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_rates_rows_cnt> on 2023-02-04 07:35:00+00:00
[2023-02-04 07:40:26,348] {standard_task_runner.py:52} INFO - Started process 6443 to run task
[2023-02-04 07:40:26,358] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'get_rates_rows_cnt', 'scheduled__2023-02-04T07:35:00+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag_test.py', '--cfg-path', '/tmp/tmpizme1b86', '--error-file', '/tmp/tmpmi4j67qx']
[2023-02-04 07:40:26,359] {standard_task_runner.py:80} INFO - Job 158: Subtask get_rates_rows_cnt
[2023-02-04 07:40:26,465] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.get_rates_rows_cnt scheduled__2023-02-04T07:35:00+00:00 [running]> on host 39df186e1726
[2023-02-04 07:40:26,567] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=get_rates_rows_cnt
AIRFLOW_CTX_EXECUTION_DATE=2023-02-04T07:35:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-04T07:35:00+00:00
[2023-02-04 07:40:26,578] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-02-04 07:40:26,601] {logging_mixin.py:115} INFO - 22
[2023-02-04 07:40:26,615] {python.py:173} INFO - Done. Returned value was: None
[2023-02-04 07:40:26,632] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=get_rates_rows_cnt, execution_date=20230204T073500, start_date=20230204T074026, end_date=20230204T074026
[2023-02-04 07:40:26,688] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-04 07:40:26,744] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
